src/core/errors.tsx

```diff
@@ -16,7 +16,17 @@ export const FloatingError = ({ message = '' }) => {
   if (hide || !message) {
     return null;
   }
-  return <Alert title={message} elevated style={{ position: 'absolute', top: 0, right: 0, width: '50%' }} />;
+  return (
+    <Alert
+      title={message}
+      elevated
+      style={{ position: 'absolute', top: 0, right: 0, width: '50%' }}
+      severity={severity as AlertVariant}
+      onRemove={() => setHide(true)}
+    >
+      {innerMessage && <div>{innerMessage}</div>}
```

Comment:
```suggestion
      {innerMessage && <span>{innerMessage}</span>}
```

nit: avoid `div` tags and use the respective tags (unless grafana doesn't allow that ofc)